packages/atomic/src/components/atomic-search-parameter-serializer/atomic-search-parameter-serializer.tsx

```diff
@@ -0,0 +1,44 @@
+import {Component, State} from '@stencil/core';
+import {
+  buildSearchParameterManager,
+  buildSearchParameterSerializer,
+  Unsubscribe,
+} from '@coveo/headless';
+import {
+  Initialization,
+  InterfaceContext,
+} from '../../utils/initialization-utils';
+
+@Component({
+  tag: 'atomic-search-parameter-serializer',
+  shadow: false,
+})
+export class AtomicSearchParameterSerializer {
```

Comment:
I don't think this should be a component. I would see it more as an option on the atomic-search-interface, that you can decide to enable or not as an end user.

I would also be against adding any component which would not be purely visual, or associated with the markup at that location in the page.

Or at least, to not try to repeat this pattern, which I think was a mistake with JSUI (having "invisible" components that act globally on the search interface)

Reply:
Indeed, in the scrum I was between a component and a prop on the search-interface. I think the latter is ultimately easier for end-users,